src/ovp_pipeline/live_concept_fileops.py

```
@@ -461,6 +461,135 @@ def patch_live(
+    """
+    if section_title not in AGENT_OWNED_SECTIONS:
+        raise ValueError(
+            f"section {section_title!r} is not in the agent-owned allowlist "
+            f"{AGENT_OWNED_SECTIONS}; agent body editor refuses to write here"
+        )
+    with acquire_lock(path):
+        text = _read_file(path)
+        if not text:
+            raise ValueError(
+                f"cannot patch_agent_section: {path} does not exist"
+            )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)